src/commands/BootstrapCommand.js

```diff
@@ -28,16 +42,11 @@ export default class BootstrapCommand extends Command {
     this.progressBar.init(this.packages.length);
     this.logger.info("Linking all dependencies");
 
-    const ignore = this.flags.ignore || this.repository.bootstrapConfig.ignore;
-
-    // Get a filtered list of packages that will be bootstrapped.
-    const todoPackages = PackageUtilities.filterPackages(this.packages, ignore, true);
-
-    // Get a trimmed down graph that includes only those packages.
-    const filteredGraph = PackageUtilities.getPackageGraph(todoPackages);
```

Comment:
this is slightly unrelated to the PR, but i moved the package filtering to the `initialize` method to mimic the behavior of `lerna run` for consistency.